web/api/api.go

```diff
@@ -31,19 +31,19 @@ type MetricsService struct {
 }
 
 // RegisterHandler registers the handler for the various endpoints below /api.
-func (msrv *MetricsService) RegisterHandler() {
+func (msrv *MetricsService) RegisterHandler(pathPrefix *string) {
```

Comment:
It's uncommon to pass a string as a pointer, as they are essentially passed by reference anyways (and immutable). So just `string` here.